samples/hello-js/README.md

```diff
@@ -0,0 +1,10 @@
+Hello, JavaScript!
+==================
+
+This sample demonstrates a simple JavaScript application with a Kotlin based build script.
+
+Simply import the Gradle project by pointing to the `setting.gradle` file.
+
+Build the project with `./gradlew clean build` to create the JavaScript-File (output is folder `web`).
+
+Open the `index.html` and check the Browser's console for output.
```

Comment:
The `index.html` is currently stored directly in the sample's root folder, so `./index.html`. Do you see this problematic?

Reply:
I'd move the `index.html` from the sample's root dir into `src/main/resources`. To copy it into `build/web/` I'd configure `the<JavaPluginConvention>().sourceSets["main"].output.resourcesDir = File("$buildDir/web/")`. It seems that I have to use `the<JavaPluginConvention>()` and not just `java`... What do you think about this?

Reply:
Moving it to `src/main/resources` and adding it to the `stageWeb` copy task should be enough.

No need to configure any source set.

BTW, you can't use the `java` static accessor here because the Kotlin JS plugin is not applied using the `plugins {}` block, see https://youtrack.jetbrains.com/issue/KT-20156